README.md

```diff
@@ -12,14 +12,29 @@ PySyft has also been explained in video form by [Siraj Raval](https://www.youtub
 
 ## Pre-Installation
 
-Optionally, we recommend that you install PySyft within the [Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/overview.html) virtual environment. If you are using Windows, I suggest installing [Anaconda and using the Anaconda Prompt](https://docs.anaconda.com/anaconda/user-guide/getting-started/) to work from the command line.
+Optionally, we recommend that you install PySyft within the
+[Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/overview.html)
+virtual environment, for its simplicity in installation. If you are using
+Windows, we suggest installing [Anaconda and using the Anaconda
+Prompt](https://docs.anaconda.com/anaconda/user-guide/getting-started/) to
+work from the command line.
 
 ```bash
 conda create -n pysyft python=3
 conda activate pysyft # some older version of conda require "source activate pysyft" instead.
 conda install jupyter notebook
 ```
 
+Another alternative is to use a plain python virtual environment. That is
```

Comment:
can we refer to these as python venvs ? That being said I could be convinced otherwise (plain python virtual environement sounds weird to me)

Reply:
Changed in both places where "plain" was used.